assignment.js

```diff
@@ -32,6 +32,13 @@ console.log('Problem 1:')
 
 // Add your code below this line
 
+let multiply = num => {
+    let baseNumber = 0
+    for (let i = 1; i < 12; i++){
+    console.log(baseNumber = num * i) 
+}
```

Comment:
@SOyeniyi801 be careful with your indentation here, this line should be indented one level to the left.